.github/workflows/deploy.yml

```
@@ -0,0 +1,78 @@
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: "pages"
+  cancel-in-progress: false
+
+jobs:
+  build:
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
```

actionlint (1.7.4)

property "deployment" is not defined in object type {detect-package-manager: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)